boxoffice/assets/js/views/router.js

```diff
@@ -36,6 +40,14 @@ export const Router = Backbone.Router.extend({
   item_collection: function(ic_id) {
     ItemCollectionView.render({ic_id});
   },
+  new_item_collection: function(org_name){
+    OrgView.render({org_name});
```

Comment:
Is `OrgView` component required here since on successful creation of an item collection, org page is fetched again and loaded.
Also there is a delay in the New item collection (/admin/o/hasgeek/ic/new) page,  the `OrgView` is loaded before before `ItemCollectionNewView` which can lead to confusion for the user